mytoyota/models/lock_status.py

```diff
@@ -70,7 +70,12 @@ def closed(self) -> Optional[bool]:
     @property
     def locked(self) -> Optional[bool]:
         """If the door is locked."""
-        return _get_status(self._status, status="carstatus_locked")
+        if _get_status(self._status, status="carstatus_locked") is True:
+            return True
+        if _get_status(self._status, status="carstatus_unlocked") is False:
+            return False
+        else:
+            return None
```

Comment:
I'm not understanding the logic here and I don't have a car with such functionality to check.
- I understand the first if statement `carstatus_locked` is `True` means Locked.
- However, `carstatus_unlocked` being `False` would also imply Locked, as its Not unlocked? Is it really like this?

Reply:
Yes, I tested it a few times yesterday with my car. The Toyota API is really very strange here.